### PR TITLE
Create docs for the metrics tab

### PR DESCRIPTION
**Pull request by alanwguo**
Signed-off-by: Alan Guo <aguo@anyscale.com>

## Why are these changes needed?
Adds information about the new metrics tab in the ray dashboard. Links to instructions on how to set it up.

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by rkooo567** [comment]
Can you update the PR description?

File: doc/source/ray-core/ray-dashboard.rst [hunk 1]
```diff
@@ -103,6 +103,18 @@ The log viewer provides various search functionality to help find the log messag
...
     :align: center
 
+Metrics View
+~~~~~~~~~~~~~
+The metrics view lets you view visualizations of the time series metrics emitted by Ray. It requires that prometheus and grafana is running for your cluster.
+Instructions about that can be found :ref:`here <ray-metrics>`.
+
+You can select the time range of the metrics in the top right corner. The graphs refresh automatically every 15 seconds.
+
+There is also a convenient button to launch grafana from the dashboard. The Grafana UI provides additional customizability of the charts.
```
Comment (rkooo567) [inline doc/source/ray-core/ray-dashboard.rst:113]: Maybe we should call it "open a Grafana dashboard" instead of "launch"?

**Review by rkooo567 — APPROVED** [review]

**Comment by rkooo567** [comment]
Is this based on the old commit? Seems like it is not the latest doc format https://ray--29414.org.readthedocs.build/en/29414/

**Comment by alanwguo** [comment]
> Is this based on the old commit? Seems like it is not the latest doc format https://ray--29414.org.readthedocs.build/en/29414/

merged with master

**Review by ericl — APPROVED** [review]

**Comment by ericl** [comment]
Merged, thanks!

**Pull request merged**